pkg/web_app/lib/src/account.dart

```diff
@@ -147,3 +156,39 @@ class _AccountNavWidget {
     }
   }
 }
+
+class _PkgAdminWidget {
+  Element _root;
+  void init(Element root) {
+    if (!pageData.isPackagePage) return;
+    _root = root;
+    _root.append(elem('h3', classes: ['title'], text: 'Admin'));
+    final discontinueToggle = elem(
```

Comment:
Aren't we also talking about moving this to a separate tab...

Reply:
Yes, but not in this PR.